kuma/settings/common.py

```diff
@@ -34,9 +34,15 @@ def __call__(self, value):
 
 ROOT = dirname(dirname(dirname(os.path.abspath(__file__))))
 
-ADMINS = config('ADMIN_EMAILS',
-                default='mdn-dev@mozilla.com',
-                cast=TupleCsv())
+ADMINS = zip(
+    config('ADMIN_NAMES', default='MDN devs', cast=Csv()),
+    config('ADMIN_EMAILS', default='mdn-dev@mozilla.com', cast=Csv())
+)
+CELERYD_MAX_TASKS_PER_CHILD = config(
+    'CELERYD_MAX_TASKS_PER_CHILD',
+    default=0,
+    cast=int
+)
```

Comment:
The default needs to be ``None``, not 0. Unfortunately, you can't use ``default=None``. This worked for me:

```
CELERYD_MAX_TASKS_PER_CHILD = config(
    'CELERYD_MAX_TASKS_PER_CHILD',
    default=0,
    cast=int
) or None
```

Reply:
Thanks @jwhitlock!

Reply:
Oh, while you are changing it - please move it down to https://github.com/escattone/kuma/blob/9cdce16cc5e844f0674518abc5275d312ef84f46/kuma/settings/common.py#L1274 , where the other ``CELERYD_*`` settings are.